src/main/js/wiki/components/BranchDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import injectSheet from "react-jss";
+import classNames from "classnames";
+import { orderBranches } from "../../repository/components/BranchOverview";
+
+const styles = {
+  dropdown: {
+    "background-color": "white",
+    border: "#777777",
+    color: "#777777",
+    padding: "6px 12px",
+    fontSize: "16px",
+    marginLeft: "auto"
+  }
+};
+
+export type Branch = {
+  name: string;
+  link: string;
+};
+
+type Props = {
+  branches: Branch[];
+  currentBranch: string;
+  classes: any;
+};
+
+export const buildBranchUrl = (oldBranchName: string, newBranchName: string) => {
+  const url = window.location.href;
+  return url.replace(oldBranchName, newBranchName);
+};
+
+class BranchDropdown extends React.Component<Props> {
+  constructor(props) {
+    super(props);
+    console.log(props);
```

Comment:
Development artefact?

Reply:
Yes, I deleted the console.log(props) line.